Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1824,6 +1850,17 @@ extension BrowserViewController: ReaderModeDelegate, UIPopoverPresentationContro
     }
 }
 
+// MARK: - UIPopoverPresentationControllerDelegate
+
+extension BrowserViewController: UIPopoverPresentationControllerDelegate {
+    func popoverPresentationControllerDidDismissPopover(popoverPresentationController: UIPopoverPresentationController) {
+        displayedPopoverController = nil
```

Comment:
Ah you're right - can't weakify the `updateDisplayedPopoverProperties` property. I'd just change `displayedPopoverController` to weak and leave the delegate implementation in there to nil out `updateDisplayedPopoverProperties`.